libs/deps/merch-mnemonic-list.js

```diff
@@ -0,0 +1,48 @@
+// branch: MWPW-142267 commit: db56fa6d1f49aa580f3da94842ffb9e73516ee18 Mon, 08 Jul 2024 21:06:50 GMT
```

Comment:
since this PR was already merged in mas repository these files should now be regenerated from /main branch

Reply:
Not sure why it still says branch is different than main but these were generated from main